databricks.json

```diff
@@ -0,0 +1,3602 @@
+{
+    "appid": "6dcc0a06-ceb4-4412-834e-b8576a7486a6",
+    "name": "Databricks",
+    "description": "This app supports investigative and data manipulation actions in Databricks",
+    "publisher": "Splunk",
+    "package_name": "phantom_databricks",
+    "type": "investigative",
+    "license": "Copyright (c) 2024 Splunk Inc.",
+    "main_module": "databricks_connector.py",
+    "app_version": "1.0.0",
+    "utctime_updated": "2022-02-04T18:14:51.000000Z",
+    "product_vendor": "Databricks",
+    "product_name": "Databricks",
+    "product_version_regex": ".*",
+    "min_phantom_version": "6.2.0",
+    "python_version": "3",
+    "fips_compliant": true,
+    "latest_tested_versions": [
+        "Cloud API, June 28, 2023"
```

Comment:
Is this really the last time we tested

Reply:
nope, we tested yesterday.... updating that.